src/deployments/bb-promster.json

```diff
@@ -0,0 +1,74 @@
+{
+  "apiVersion": "apps/v1",
+  "kind": "Deployment",
+  "metadata": {
+    "name": "test-bb-promster-example"
+  },
+  "spec": {
+    "replicas": 1,
+    "selector": {
+      "matchLabels": {
+        "name": "test-bb-promster-example"
+      }
+    },
+    "template": {
+      "metadata": {
+        "labels": {
+          "name": "test-bb-promster-example"
+        }
+      },
+      "spec": {
+        "containers": [
+          {
+            "name": "test-bb-promster-example",
+            "image": "labbsr0x/bb-promster:latest",
+            "env": [
+              {
+                "name": "REGISTRY_ETCD_URL",
+                "value": "http://etcd.default.svc.cluster.local:2379"
+              },
+              {
+                "name": "REGISTRY_SERVICE",
+                "value": "example"
+              },
+              {
+                "name": "REGISTRY_ETCD_BASE",
+                "value": "/services"
+              },
+              {
+                "name": "REGISTRY_SERVICE",
+                "value": "example"
+              },
+              {
+                "name": "REGISTRY_ETCD_BASE",
+                "value": "/services"
+              },
+              {
+                "name": "BB_PROMSTER_LEVEL",
+                "value": "1"
+              },
+              {
+                "name": "ETCD_URLS",
+                "value": "http://etcd.default.svc.cluster.local:2379"
+              },
+              {
+                "name": "SCRAPE_ETCD_PATH",
+                "value": "/services/example"
+              },
+              {
+                "name": "REMOTE_WRITE_URL",
+                "value": "http://nginx.cortex.svc.cluster.local/api/prom/push"
```

Comment:
@gutorc92, I'm worried about these fixed values here. What are you using this `bb-promster.json` for?

Reply:
bb-promster.json is a template for a deployment on Kubernetes.